### PR TITLE
7.4.1 release

### PR DESCRIPTION
**Pull request by jmlrt**
does what the tin say

### CODE REVIEW

**Comment by jmlrt** [comment]
jenkins test this please

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,34 @@
 # Changelog
 
+## 7.4.1 - 2019/10/22
+
+* 7.4.1 as the default stack version
+* 6.8.4 as 6.x tested version
+* Helm 2.15.1 support in [#309](https://github.com/elastic/helm-charts/pull/338) [@jmlrt](https://github.com/jmlrt)
```
Comment (Conky5) [inline CHANGELOG.md]:
```suggestion
* Helm 2.15.1 support in [#338](https://github.com/elastic/helm-charts/pull/338) [@jmlrt](https://github.com/jmlrt)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,34 @@
 # Changelog
 
+## 7.4.1 - 2019/10/22
```
Comment (Conky5) [inline CHANGELOG.md]:
```suggestion
## 7.4.1 - 2019/10/23
```

**Comment by jmlrt** [comment]
fixed changelog and removed logstash files commited by mistake

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,34 @@
 # Changelog
 
+## 7.4.1 - 2019/10/23
+
+* 7.4.1 as the default stack version
+* 6.8.4 as 6.x tested version
+* Helm 2.15.1 support in [#309](https://github.com/elastic/helm-charts/pull/309) [@jmlrt](https://github.com/jmlrt)
```
Comment (Conky5) [inline CHANGELOG.md]: I think this should be https://github.com/elastic/helm-charts/pull/338 ?

Comment (jmlrt) [reply CHANGELOG.md]: yes sorry 🤦‍♂

Comment (Conky5) [reply CHANGELOG.md]: No worries

**Pull request merged**